notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11675,6 +11675,50 @@ opposition to authority [Neo's boss at the software company inside the Matrix co
 the need for freedom [It was mentioned about how life inside the Matrix did not constitute a free mode of existence for humans.]
 
 
+movie: The Thirteenth Floor (1999)
+==================================
+
+:: Title
+The Thirteenth Floor
+
+:: Description
+The Thirteenth Floor is a 1999 science fiction neo-noir film written and
+directed by Josef Rusnak, and produced by Roland Emmerich through his
+Centropolis Entertainment company. It is loosely based upon Simulacron-3
+(1964), a novel by Daniel F. Galouye, and a remake of the German tv-film World
+on a Wire (1973).
+
+:: Date
+1999-04-16
+
+:: References
+https://en.wikipedia.org/wiki/The_Thirteenth_Floor
+
+:: Ratings
+5 <paul>
+
+:: Related Stories
+movie: World on a Wire (1973)
+
+:: Choice Themes
+speculative virtual reality [The film follows Douglas Hall as he pieces together that the world of his experience is all just a big computer simulation.]
```

Comment:
Does "the universe is a simulation" then imply this?

Reply:
I guess intuitively we have reserved VR for simulations that are man-made and accessed more or less at will.

Reply:
I used **the universe is a simulation** there in choice themes without hesitation. You have some reservations?

About **speculative virtual reality**, as I recall we used to have **virtual reality** and something like **simulated reality**, but decided to merge them both together.

Reply:
According to your comment, VR is a theme because the universe is a simulation. That would mean that the universe is a simulation should have the VR theme as a parent. A little weird but conceivable. If not, you should improve the motivation here. Something to discuss on Wednesday?

Reply:
Possibly we need to have a look at the definition for the VR theme too